index.html

```diff
@@ -14,12 +14,30 @@
   <header>
     <h1>How do Collaboration work?</h1>
   </header>
-<nav>
-  <h2>Nav</h2>
-</nav>
-<section>
-  <h3>Section</h3>
-</section>
+  <nav>
+    <h2>Nav</h2>
+  </nav>
+  <div class="container">
+    <div>
+      <section>
+        <h3>Section</h3>
+      </section>    
+      <article>
+        <h1>An amazing article about a collaborative project</h1>
+        <p>Hey all! Look we created our first integrate project using branches and pull requests. It's really exciting  to work together.<br>
+              Nevertheless, it's not something easy. We have to understand each other and make some conventions about how to name branches, what everyone have to do; so on and so on...
+        </p>
+        <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nemo, unde et. Soluta, mollitia odit sint accusantium iste molestiae sequi, nemo dolores magni totam repellendus quisquam quas id libero laudantium hic?</p>
+      </article>
+    </div> 
```

Comment:
can you remove this div. and can you remove the p. there is no need text.

Reply:
I think it changes nothing to the exercice